src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -150,27 +150,30 @@ export class Create extends Setup {
     }
 
     private async executeAndroidDeviceCreate(): Promise<any> {
```

Comment:
Not sure if we'd do it as part of this exercise, but: should we make the return type `Promise<boolean>` here? That's what `AndroidSDKUtils.createNewVirtualDevice()` returns, which is what we're ultimately returning.